src/main/java/org/sagebionetworks/bridge/models/studies/Demographic.java

```diff
@@ -0,0 +1,129 @@
+import javax.persistence.FetchType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.persistence.Table;
+
+import org.sagebionetworks.bridge.models.BridgeEntity;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+/**
+ * Represents a demographic category and value(s) in that category for a
+ * specific user. Contains multiple DemographicValues.
+ */
+@Entity
+@Table(name = "Demographics")
+public class Demographic implements BridgeEntity {
+    @Nonnull
+    @Id
+    String id;
+
+    @JsonIgnore
+    @ManyToOne
+    @JoinColumn(name = "demographicUserId")
+    DemographicUser demographicUser;
+
+    @Nonnull
+    @JsonIgnore // used as key to parent DemographicUser map instead
+    String categoryName;
+
+    @Nonnull
+    private boolean multipleSelect = true;
+
+    @Nonnull
+    @ElementCollection(fetch = FetchType.EAGER)
+    @CollectionTable(name = "DemographicsValues", joinColumns = @JoinColumn(name = "demographicId", referencedColumnName = "id"))
+    private List<DemographicValue> values;
+
+    @JsonInclude(JsonInclude.Include.NON_NULL)
+    private String units;
+
+    public Demographic(String id, DemographicUser demographicUser, String categoryName, boolean multipleSelect,
+            List<DemographicValue> values, String units) {
+        this.id = id;
+        this.demographicUser = demographicUser;
+        this.categoryName = categoryName;
+        this.multipleSelect = multipleSelect;
+        this.values = values;
+        this.units = units;
+    }
+
+    public Demographic() {
+    }
+
+    public String getId() {
+        return id;
+    }
+
+    public void setId(String id) {
+        this.id = id;
+    }
+
+    public DemographicUser getDemographicUser() {
+        return demographicUser;
+    }
+
+    public void setDemographicUser(DemographicUser demographicUser) {
+        this.demographicUser = demographicUser;
+    }
+
+    public String getCategoryName() {
+        return categoryName;
+    }
+
+    public void setCategoryName(String categoryName) {
+        this.categoryName = categoryName;
+    }
+
+    public boolean isMultipleSelect() {
+        return multipleSelect;
+    }
+
+    public void setMultipleSelect(boolean multipleSelect) {
+        this.multipleSelect = multipleSelect;
+    }
+
+    public List<DemographicValue> getValues() {
+        return values;
+    }
+
+    @JsonProperty("values")
+    public void setValues(List<DemographicValue> values) {
+        if (values != null) {
+            // remove null
+            for (ListIterator<DemographicValue> iter = values.listIterator(); iter.hasNext();) {
+                DemographicValue next = iter.next();
+                if (next == null || next.getValue() == null) {
+                    iter.remove();
+                }
+            }
+        }
```

Comment:
> null value in DemographicValue

For this format I don't have a custom deserializer; Jackson constructs this type automatically, and if a value is null then it should make a null DemographicValue (which I added a test for) instead of a DemographicValue containing null. Since it isn't constructed explicitly anywhere and I don't think there's any possible input that can result in this, I couldn't think of a way to test this case directly, so I added the test on this Demographic.setValues() method itself.